benchmarks/ssr/common.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const compiler = require('../../dist/compiler.common.js')
+const self = (global || root)
+
+self.performance = {
+  now: function () {
+    var hrtime = process.hrtime()
+    return ((hrtime[0] * 1000000 + hrtime[1] / 1000) / 1000)
+  }
+}
+
+function generateGrid (rowCount, columnCount) {
+  var grid = []
+
+  for (var r = 0; r < rowCount; r++) {
+    var row = { id: r, items: [] }
+    for (var c = 0; c < columnCount; c++) {
+      row.items.push({ id: (r + '-' + c) })
+    }
+    grid.push(row)
+  }
+
+  return grid
+}
+
+const gridData = generateGrid(100000, 100)
+
+var perfMixin = {
+  computed: {
+    performance: {
+      cached: false,
+      get: function () {
+        return (self.performance.now() - self.s).toFixed(2)
+      }
+    }
+  }
+}
+
+var gridComponent = {
+  template: '<div><h1>{{ performance }}ms</h1><my-table></my-table></div>',
+  mixins: [perfMixin],
+  components: {
+    myTable: {
+      data: function () {
+        return {
+          grid: gridData
+        }
+      },
+      template: '<table width="100%" cellspacing="2"><row v-for="row in grid"></row></table>',
```

Comment:
actually the benchmark is broken because you are not passing in the `:row` prop at all :/